src/uu/env/src/env.rs

```diff
@@ -87,6 +93,64 @@ fn parse_program_opt<'a>(opts: &mut Options<'a>, opt: &'a OsStr) -> UResult<()>
     }
 }
 
+fn parse_signal_value(signal_name: &str) -> UResult<usize> {
+    let signal_name_upcase = signal_name.to_uppercase();
+    let optional_signal_value = signal_by_name_or_value(&signal_name_upcase);
+    let error = USimpleError::new(125, format!("{}: invalid signal", signal_name.quote()));
+    match optional_signal_value {
+        Some(sig_val) => {
+            if sig_val == 0 {
+                Err(error)
+            } else {
+                Ok(sig_val)
+            }
+        }
+        None => Err(error),
+    }
+}
+
+fn parse_signal_opt<'a>(opts: &mut Options<'a>, opt: &'a OsStr) -> UResult<()> {
+    if opt.is_empty() {
+        return Ok(());
+    }
+    #[cfg(unix)]
+    let signals: Vec<&'a OsStr> = opt
+        .as_bytes()
+        .split(|&b| b == b',')
+        .map(OsStr::from_bytes)
+        .collect();
+    #[cfg(not(unix))]
+    let signals: Vec<&'a OsStr> = opt
+        .as_encoded_bytes()
```

Comment:
Why can't `as_encoded_bytes` be used on unix-platforms? The [documentation](https://doc.rust-lang.org/std/ffi/struct.OsStr.html#method.as_encoded_bytes) looks like it should work on all platforms.

Reply:
Sorry! Guess I missed that it's been fixed though